src/Microsoft.Identity.Client/User.cs

```diff
@@ -47,37 +46,34 @@ internal User(User other)
             IdentityProvider = other.IdentityProvider;
         }
 
+        public User(string identifier, string displayableId, string name, string identityProvider)
```

Comment:
Changed the _Create_ method to be a public constructor.
I'm guessing that the _identifier_ value is required, so I've moved it first (it's usual to have required parameter appearing before optional parameters).